tests/cli/test_compose_sanity.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+. /usr/share/beakerlib/beakerlib.sh
+
+CLI="./src/bin/composer"
+
+
+rlJournalStart
+    rlPhaseStartTest "compose types"
+        rlAssertEquals "lists all supported types" \
+                "`$CLI compose types | sort | xargs`" "ext4-filesystem live-iso partitioned-disk qcow2 tar"
```

Comment:
These are always sorted and I would prefer it to fail every time there is a change. Otherwise we can just go without this assert and simply make sure the output isn't an empty string.